tools/topology/platform/intel/intel-generic-dmic-kwd.m4

```diff
@@ -57,10 +57,18 @@ dnl     time_domain, sched_comp)
 # Passthrough capture pipeline using max channels defined by CHANNELS.
 
 # Set 1000us deadline on core 0 with priority 0
+ifdef(`DMICPROC_FILTER1', `define(PIPELINE_FILTER1, DMICPROC_FILTER1)', `undefine(`PIPELINE_FILTER1')')
```

Comment:
@singalsu the change looks good, but what did you mean to day here ? "If the pipelines are used for any other purpose than pipelines ..."

Reply:
Thanks, messed up sentence, I need to fix the commit message. I meant that those pipeline macros can't be used for other than DMIC endpoint.

Reply:
It's fixed now!